src/platform_impl/web/async/wrapper.rs

```diff
@@ -81,3 +83,29 @@ impl<V, S: Clone + Send, E> Clone for Wrapper<V, S, E> {
         }
     }
 }
+
+impl<V, S: Clone + Send, E> Eq for Wrapper<V, S, E> {}
```

Comment:
Generally `Eq`/`Ord` go after their `Partial` impls.

Reply:
I generally do this alphabetically in the Web backend.
So I would like to stick to that.

I can make a follow-up PR where I align all `derive`s and implementations in the whole crate to do `PartialEq/PartialOrd` before `Eq/Ord` and generally have the same order throughout the crate if you like.

Reply:
I'd just suggest to do that for the new stuff since it's just pointless, unless we have a formatter support.

I won't block on the current order either, just noticed that it's usually other way around (you can also `derive(Eq)` if you have `impl PartialEq`).

Reply:
No unfortunately no formatter support ...
I will make a PR only for Web then, because Web follow alphabetical order so far.